internals/cli/cli.go

```diff
@@ -289,6 +289,20 @@ func Run(options *RunOptions) error {
 	if options == nil {
 		options = &RunOptions{}
 	}
+	if options.PebbleDir == "" {
+		options.PebbleDir = cmd.DefaultDir
+	}
+	log := options.Logger
+	if log == nil {
+		log = logger.New(os.Stderr, fmt.Sprintf("[%s] ", cmd.ProgramName))
+	}
+	logger.SetLogger(log)
+	if options.ClientConfig == nil {
+		options.ClientConfig = &client.Config{}
+	}
+	if options.ClientConfig.Socket == "" {
+		options.ClientConfig.Socket = filepath.Join(options.PebbleDir, ".pebble.socket")
+	}
```

Comment:
I propose a cleanup. Lets do all the default values on the inside, and all the inputs (including environment var) on the outside.